server/README.md

```diff
@@ -0,0 +1,13 @@
+# Python server
+Usage:
+
+`python server.py [-p PORT] [-a ADDRESS| address]`
```

Comment:
This will be hard to maintain when we add more CLI flags and is usually available on -h anyway.
It is better to replace this with an actual example that can be copy-pasted and run.